src/cocoa/toga_cocoa/widgets/tree.py

```diff
@@ -118,14 +133,14 @@ def create(self):
         # Add the layout constraints
         self.add_constraints()
 
-    def insert_node(self, node):
-        node._impl = TogaNodeData.alloc().init()
-        node._impl.node = node
+    # def insert_node(self, node):
```

Comment:
We think this is no longer necessary because the data source captures all of it, yes?

Reply:
It's more about the Cocoa widget; we no longer need the lookup mechanism, because the Cocoa delegate can find the TogaData objects directly from the Row/Node object provided by the DataSource.